app/Api/V1/Domain/Problem/Param/ReadProblemParam.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Api\V1\Domain\Problem\Param;
+
+
+class ReadProblemParam
+{
+    const ATTRIBUTE_LIMIT = 'limit';
+
+    const QUERY_PARAMS = [
+        ReadProblemParam::ATTRIBUTE_LIMIT,
+    ];
+
+    protected $data;
+
+    public function __construct()
+    {
+        $this->data[ReadProblemParam::ATTRIBUTE_LIMIT] = null;
+    }
+
+    public function fromArray(array $array): void
+    {
+        $this->data[ReadProblemParam::ATTRIBUTE_LIMIT] = $array[ReadProblemParam::ATTRIBUTE_LIMIT];
+    }
+
+    public function getLimit(): int
```

Comment:
Limit could be null so this function return type should not be `int`, you could use `?int`